content/content-strategy/identify-business-needs/who-to-talk/business-alignment-workshop/text4.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: text/textblock
+---
+The workshop should start with introductions and an icebreaker.  
+
+If you have a clear idea of the vision, mission and goals of the agency, show participants and get joint agreement. 
+
+If you anticipate disagreement, get them to write up the vision on post-it notes. Group the post-its and get agreement. Do the same for the mission statements. You should co-design statements together. 
+
+It might be easier to get into smaller groups if you need to brainstorm missions. 
+
+The goal of the workshop is to get agreed vision, mission and goals of your agency. 
+
+**Props can help facilitate the workshop:**  
```

Comment:
This isn't consistent with anything on other topic pages. Reads a bit like a tip or takeaway. If so, please move to a new partial with the callout style. Otherwise style as a heading.